### ISSUE TITLE
Support for Electra iOS 11.2-11.3.1

### ISSUE THREAD

**Issue by ghost** [BishopFox/bfinject#36]
Can this be updated to support the recent Electra jailbreak?

https://coolstar.org/electra/

Currently I'm getting the following error: `[!] Unknown jailbreak. Aborting.`

**Comment by level-zer0** [comment]
I haven't used the Electra iOS 11.2-11.3.1 jailbreak yet. But based on the error, is the bfinject script pointing to the correct directory? You might have to edit the path outlined below as well as others accordingly.

![screen shot 2018-07-09 at 1 12 49 pm](https://user-images.githubusercontent.com/41021765/42468247-eddf3282-8379-11e8-9dc6-cd2e3cb7902c.png)

**Comment by svedm** [comment]
@bamtan I'm created quick-fix script for Electra 11.3.1, but I'm too lazy for creating PR
https://gist.github.com/svedm/93bb792ddb22fedf5ab487b1dbf2e897

**Comment by ghost** [comment]
@svedm thanks for this fix! I can confirm it's working on my 8+ 11.3.1.
edit: upon testing, apps crash on startup after resigning

**Comment by fahasogato** [comment]
Hello guys somebody can help me? I trying to inject cycript but it said unknown jailbreak aborting. I write this command:
bash bfinject -p 3094 -L cycript                                   
[!] Unknown jailbreak. Aborting.

**Comment by fahasogato** [comment]
Someone can help me ? It said no Team id

**Comment by thelittletrain171** [comment]
@svedm testing with the reddit app with iphone8 on 11.3.1. I get this error when running `/var/mobile/Documents/bfinject root# bash bfinject -P Reddit -L decrypt`  
  
bfinject: line 43: wc: command not found  
[!] "Reddit" was not uniquely found, please check your criteria.

**Comment by syto203** [comment]
[MJavad's](https://github.com/MJavad/bfinject) fork work under ios 11-11.4b3 even with tweaks turned on. you just need to place it under `/ , /private /bin /sbin`. `/var` wont work bec. it doesnt allow code execution . However it doesnt sign correctly but you can do it manually via `ldid -S path/to/binary` or use this automated script on the ipa [signer.sh](https://pastebin.com/raw/MaF8FXqB). usage `bash signer.sh path/to/ipa`

**Comment by rootbsd** [comment]
It works perfectly on 11.3.1 without tweaks. It fails with tweaks enabled.

**Comment by syto203** [comment]
Yes. MJavad however found a way to run it with tweaks enabled. I’ve tested it works up to 11.4b3

**Comment by rootbsd** [comment]
I created the PR for you.

**Comment by orxelm** [comment]
> Someone can help me ? It said no Team id

me too
